src/components/Pill/PillWrapper.tsx

```diff
@@ -3,57 +3,45 @@ import styled, { css } from 'styled-components';
 import PropTypes from 'prop-types';
 
 import { SpaceSizes } from '../../theme';
-import { getColor, getRadii } from '../../utils';
+import { getRadii } from '../../utils';
 import { Inline, Padbox } from '../layout';
 import { PaddingTypes } from '../layout/Padbox/Padbox.enums';
-import { PillSizes, PillVariants } from './Pill.enums';
+import { PillColors as PillColorsEnums } from './Pill.enums';
 import { PillWrapperProps, StyledPillWrapperProps } from './Pill.types';
-
-const PillSolid = css`
-  background-color: ${getColor('neutral.300')};
-`;
-
-const PillOutline = css`
-  background-color: ${getColor('neutral.0')};
-  box-shadow: inset 0 0 0 1px ${getColor('neutral.500')};
-`;
-
-const pillVariants = {
-  [PillVariants.solid]: PillSolid,
-  [PillVariants.outline]: PillOutline,
-};
+import { PillColors } from './PillColors';
 
 const StyledPillWrapper = styled(Padbox)<StyledPillWrapperProps>`
   display: inline-block;
   min-width: 0;
   border-radius: ${getRadii('default')};
-  ${({ $variant }) => pillVariants[$variant]};
-
-  ${({ $isClickable }) =>
-    $isClickable &&
+  ${({ $color }) =>
     css`
-      cursor: pointer;
-
+      background-color: ${PillColors.pill[$color]};
       &:hover,
       &:focus {
-        background-color: ${getColor('primary.50')};
+        background-color: ${PillColors.pillHover[$color]};
         outline: none;
       }
```

Comment:
This should go a bit lower into `$isClickable` condition. Right now even pills that are not clickable have hover state